lambdas/handlers/create_document_reference_handler.py

```diff
@@ -63,10 +64,10 @@ def processing_event_details(event):
             f"Invalid json in body: {str(e)}",
```

Comment:
Might be worth plugging the enum value here also to avoid log messages and exception messages getting out of sync if the messaes values were to change in future. e.g `f"Invalid json in body: {str(e)}"` -> `f"{LambdaError.CreateDocPayload}: {str(e)}"`